### DISCUSSION TITLE
[Idea] Add a way to track mounts for collecting

### DISCUSSION THREAD

**Discussion by seadub** [exochron/MountJournalEnhanced#170] [Ideas]
While perusing the list of mounts and not realizing I had "Not Collected" mounts visible, I saw one I thought I would want to work towards. I see that I can "Set Note" when right-clicking a mount, but thought it would be helpful to add it to something like a farm list.

Loving the addon so far, thanks for all you've done!

**Comment by exochron** [comment]
It should be already possible with current features.
You can write a note and search for that. You can also save that search in one of the quick filters.
